src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$box-size: 100px;
+$margin-left: 103px;
+$month-lengths: (
+  28: 29,
+  29: 30,
+  30: 31,
+  31: 32,
+);
```

Comment:
The `$month-lengths` map contains incorrect values. Months do not have lengths of 32 days. The map should reflect the actual number of days in each month. For example, `30: 31` means a month with 30 days is followed by one with 31 days. Please correct the values to represent real-world data.